nimx/view.nim

```diff
@@ -210,11 +210,21 @@ proc insertSubview*(v, s: View, i: int) =
         s.viewDidMoveToWindow()
         v.didAddSubview(s)
         v.setNeedsDisplay()
+    else:
+        if i > v.subviews.len() - 1:
+            return
+        s.removeFromSuperview(false)
+        v.subviews.insert(s, i)
+        s.superview = v
+        v.didAddSubview(s)
+        v.setNeedsDisplay()
 
 proc insertSubviewAfter*(v, s, a: View) = v.insertSubview(s, v.subviews.find(a) + 1)
 proc insertSubviewBefore*(v, s, a: View) = v.insertSubview(s, v.subviews.find(a))
 proc addSubview*(v: View, s: View) = v.insertSubview(s, v.subviews.len)
 
+method dragInProcess*(v: View): bool {.base.} = false
```

Comment:
I really don't like this. Looks like a quick hack you came up with to solve your particular problem ;)